site/server/conf/evolutions/default/2.sql

```diff
@@ -0,0 +1,16 @@
+# --- !Ups
+
+DROP TABLE users;
+CREATE TABLE users (
+    id integer primary key,
+    login varchar(255) unique not null,
+    name varchar(255) not null,
+    github_id varchar(255) not null,
```

Comment:
We should try to avoid these _ leaking through Scala code and SQL if possible since it is not common practice in Scala to include such name conventions.